cylc/flow/network/schema.py

```diff
@@ -1701,13 +1708,12 @@ class WorkflowStopMode(graphene.Enum):
     """The mode used to stop a running workflow."""
 
     # NOTE: using a different enum because:
-    # * Graphene requires special enums.
     # * We only want to offer a subset of stop modes (REQUEST_* only).
 
-    Clean = cast('Enum', StopMode.REQUEST_CLEAN.value)
-    Kill = cast('Enum', StopMode.REQUEST_KILL.value)
-    Now = cast('Enum', StopMode.REQUEST_NOW.value)
-    NowNow = cast('Enum', StopMode.REQUEST_NOW_NOW.value)
+    Clean = cast('Enum', StopMode.REQUEST_CLEAN)  # type: graphene.Enum
+    Kill = cast('Enum', StopMode.REQUEST_KILL)  # type: graphene.Enum
+    Now = cast('Enum', StopMode.REQUEST_NOW)  # type: graphene.Enum
+    NowNow = cast('Enum', StopMode.REQUEST_NOW_NOW)  # type: graphene.Enum
```

Comment:
This doesn't look right. I've opened https://github.com/dwsutherland/cylc-flow/pull/22 with some fixes

Reply:
It is correct, this is part of the way the newer version of `graphql-core` handles enums "better".. 

And I'd have to test your branch, because it looks like it may reintroduce the old way of doing things.

Reply:
You're right though, it does seem a little weird .. but the mutations or validation failed with I had the `.value` in there .. will play about..

Reply:
> This is part of the way the newer version of `graphql-core` handles enums "better"..

I don't think so? In https://github.com/graphql-python/graphene/wiki/v3-release-notes#better-enum-support it talks about being able to use the `graphene.Enum` member in the resolvers rather than the member value. But the examples they give are still typical Enum patterns, e.g.:
```
class Color(graphene.Enum):
    RED = 1
    GREEN = 2
```
where the values are integers or strings. The values should not be `enum.Enum` members, as far as I can tell.

Reply:
I think the needed change for us was in the argument defaults:
```
-            default_value=BroadcastMode.Set.name,
+            default_value=BroadcastMode.Set,
```
(which is done)
Have tested your PR, makes sense, passes tests.. will get it in.